### PR TITLE
Address comments in LTS-161

### PR DESCRIPTION
**Pull request by pkubanek**
Vastly improved description of M1M3 commands. Added units to command
parameters. Dropped unused commands.

### CODE REVIEW

File: sal_interfaces/MTM1M3/MTM1M3_Commands.xml [hunk 1]
```diff
@@ -302,17 +288,21 @@
...
+        to zero the forces applied to the system by the applyActiveOpticForces
+        command.
+    </Description>
   </SALCommand>
   <SALCommand>
     <Subsystem>MTM1M3</Subsystem>
     <EFDB_Topic>MTM1M3_command_positionM1M3</EFDB_Topic>
+    <Description>
+        Change absolute M1M3 position. Position the mirror with relatively to
+        reference position specified in configuration file. New mirror position
```
Comment (rbovill) [inline sal_interfaces/MTM1M3/MTM1M3_Commands.xml:302]: Position the mirror **relative to the reference** position specified in **the** configuration file

File: sal_interfaces/MTM1M3/MTM1M3_Commands.xml [hunk 2]
```diff
@@ -302,17 +288,21 @@
...
+        will be xPosition, yPosition and zPosition and mirror will be rotate by
+        xRotation, yRotation and zRotation.
```
Comment (rbovill) [inline sal_interfaces/MTM1M3/MTM1M3_Commands.xml:304]: will be **rotated** by

**Review by rbovill — CHANGES_REQUESTED** [review]
A couple of minor, but possibly significant, changes requested.

**Comment by pkubanek** [comment]
Addressed Rob's suggestions.

**Review by rbovill — APPROVED** [review]
LGtM

**Pull request merged**